BuildTasks/ExtensionVersion/ExtensionVersion.ts

```diff
@@ -24,7 +24,15 @@ if (!usingOverride) {
 
         // Read gallery endpoint
         const galleryEndpoint = common.getMarketplaceEndpointDetails();
-        tfx.arg(["--token", galleryEndpoint.token]);
+        if (galleryEndpoint.token) {

```

Comment:
Probably we could put this logic into a common method like `common.setTfxEndpointArgs'. In this method we could call the getMarketplaceEndpointDetails and set the auth and server params.